### PR TITLE
Assignment2 /Farismhg

### PR DESCRIPTION
**Pull request by Farismhg**

### CODE REVIEW

**Comment by sayloradm** [comment]
+1 Easy to read code, README is solid. Possibly a bit more description on how to run the code.

File: problems/euler/Smallest multiple/Java/Farismhg/Multiple.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+public class Multiple
+{
+    // Euclidian algorithm for finding greatest common divisor
+    private static int GCD(int a, int b)
```
Comment (benbarn313) [inline problems/euler/Smallest multiple/Java/Farismhg/Multiple.java:4]: could maybe use comments on the working of the algorithm, not sure if necessary.

File: problems/euler/Smallest multiple/Java/Farismhg/Multiple.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            if(a < b)
+                b -= a;
+            else
+                a -= b;
+        }
+
+        return a;
+    }
+
+    public static void main(String[] args)
```
Comment (benbarn313) [inline problems/euler/Smallest multiple/Java/Farismhg/Multiple.java:17]: very concise and well commented. Cant see much need for improvement

**Review by Swingus — APPROVED** [review]
I didn't see much room for improvement, other than adding a few more comments to explain why you're doing a given step. +1

**Review by Swingus — APPROVED** [review]
image
+10

**Pull request merged**